app/controllers/api/v1/forms_controller.rb

```diff
@@ -69,6 +67,10 @@ def show_draft
     render json: form.draft_version, status: :ok
   end
 
+  def show_archived
+    render json: form.last_live_version, status: :ok
```

Comment:
What happens if the form is not archived?

Reply:
Not sure what should happen, any suggestions? Could just 404, there's no archived form by that ID etc.

Reply:
404 seems sensible, there is no archived resource for the given form id

Reply:
sorted, added a 404 case when the form isn't archived